### PR TITLE
Fix/job edit

### PR DESCRIPTION
**Pull request by AnastasiaLes**
added notifications to edit job and post job pages
fixed hourly rate error message 
fixed chat bug with undefined after search

### CODE REVIEW

**Review by KashyretsIvanna — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
    [Download and install](https://nodejs.org/en/) if it necessary.
 2. Clone this repo.
 3. Install the basic project dependencies with the command `npm install`.
-4. Create .env.local file @ apps/freelance (near env.example file). Should include variables from example file with correct   values
+4. Create .env.local file @ apps/freelance (near env.example file). Should include variables from example file with correct values
```
Comment (lyudad) [inline README.md]: why you push this file?

File: README.md [hunk 2]
```diff
@@ -15,24 +15,27 @@
...
 (in order for sync script to work user needs to get access to remote website, once, then it will always work)
 
 How to get access to remote website (all values are example, use real keys)
+
 - download/install Amazon CLI: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
-In AWS console enter:
+  In AWS console enter:
 - aws configure
-will ask AWS Access Key ID
+  will ask AWS Access Key ID
```
Comment (lyudad) [inline README.md]: same

**Review by lyudad — APPROVED** [review]

**Review by lyudad — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AnastasiaLes) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (AnastasiaLes) [reply README.md]: fixed

**Pull request merged**